api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -35,6 +35,13 @@ public String readMandrillApiKey() {
     return mandrillKeys.getString("api-key");
   }
 
+  public void copyDemoNotebook(String workspaceBucket)  {
+    Storage storage = StorageOptions.getDefaultInstance().getService();
+    BlobId demoNotebookId = storage.get(getDemosBucketName(), "demo-notebook.ipynb").getBlobId();
```

Comment:
I think we should probably check this into source control, then deploy it as a GCS artifact along with our Jupyter user initialization script. (or alternatively, skip GCS and just read it server side from the server resources and write it directly to GCS).

We already have a separate "deploy-gcs-artifacts" command which allows us to push these without a full release. There's a separate question of whether it's actually desirable to push these demo notebooks out-of-band, i.e. bypassing our normal QA process (in which case, reading it as a server resource is the most hermetic option).

Reply:
Fritz actually asked me to make sure they could be pushed out-of-band, because I believe in the long run this is going to be written by comms and scientist folks, and not by us, so they should be able to push/update them.